### ISSUE TITLE
Consider auto-versioning based on RSS feed

### ISSUE THREAD

**Issue by denysvitali** [teamatldocker/jira#135]
Maybe this is a duplicate of #91, but why don't you build the Docker image by using an argument to the Dockerfile (the Jira Version) that's automatically filled by a bash script, so that the image building process can be automated via a CI with a cronjob?
  
I'm currently doing it with denysvitali/docker-jira, more specifically in the [build.sh](https://github.com/denysvitali/docker-jira/blob/master/build.sh) script.
  
Feel free to copy it as per your need. This will avoid the hassle to manually check for a new update on Atlassian's website

**Comment by blacklabelops** [comment]
We have a version reference:

- Inside the Dockerfile
- Inside the readme.md
- Inside the release.sh

For a clear solution we should disregard all these references. Anyone in favor to just always build the latest and disregard manually maintaining references in files?

**Comment by neuhausf** [comment]
@blacklabelops Yes, please always build the latest automatically. Otherwise we depend on you to build the current version. Right now, there is 8.3.0 in production but your image points to 8.2.4 😉

**Comment by esciara** [comment]
Just for info : JIRA docker image does it (which I like less because it is less configurable than this one) through a CircleCI crin job. See https://github.com/cptactionhank/docker-atlassian-jira-software/blob/master/.circleci/config.yml . (File and scripts are reasonably well commented)

Note that it looks at the JIRA version json feed, changes the versions on the Dockerfile, builds a new docker image and runs a fair amount of tests in ruby/capybara to setup the server and test most things work as expected. If it does, it then releases the image in the docker repo.
